src/SDKs/SqlManagement/Sql.Tests/Utilities/SqlManagementTestUtilities.cs

```diff
@@ -40,6 +40,10 @@ public class SqlManagementTestUtilities
 
         public const string DefaultEuapPrimaryLocation = "East US 2 EUAP";
 
+        public const string DefaultLogin = "dummylogin";
+
+        public const string DefaultPassword = "Un53cuRE!";
```

Comment:
This is how we've been doing tests for all the SDKs, so I think it should be fixed separate from this PR.  Please create a separate github issue for this.